app/controllers/users_controller.rb

```diff
@@ -299,8 +299,8 @@ def reauthenticate
       true
     else
       wrong_password!(params[:new_email],
-                      t("users.confirm_change_email.wrong_password_html", contact_support_link: helpers.link_to(t("users.confirm_change_email.contact_support"), new_feedback_report_path)),
-                      t("users.changed_password.wrong_password"))
+                      t("users.confirm_change_email.wrong_password_html", contact_support_link: helpers.link_to(t("users.confirm_change_email.user.contact_support"), new_feedback_report_path)),
+                      t("users.changed_password.wrong_password_html", contact_support_link: helpers.link_to(t("users.changed_password.user.contact_support"), new_feedback_report_path)))
```

Comment:
I meant this here, apologies for the incorrect edit suggestion earlier

 ```suggestion
                       t("users.confirm_change_email.wrong_password_html", contact_support_link: helpers.link_to(t("users.confirm_change_email.contact_support"), new_feedback_report_path)),
                       t("users.changed_password.wrong_password_html", contact_support_link: helpers.link_to(t("users.changed_password.contact_support"), new_feedback_report_path)))
```